README.md

```diff
@@ -1 +1,214 @@
+- **[is a static module bundler for modern JavaScript applications]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git installed.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  cd To-do-list
+  git clone https://github.com/Obimbo07/To-do-list
+```
+--->
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd To-do-list
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+
+```
+  live server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+
+```npx hint .
+```
+
+### Deployment
+
+ Here is a link to the live demo of the website [Click here](https://obimbo07.github.io/To-do-list/) 
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Obimbo07](https://github.com/Obimbo07)
+- Twitter: [@Austine](https://twitter.com/austsine)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/austin-obimbo-9a613623a/)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Implement a time section to specify task duration]**
+- [ ] **[A remainder to enable user get notified about task schedule]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Obimbo07/To-do-list/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project , please show your support by:
+
+- Contributing to growth and improvement of our code base
+- Give us a 5 Star
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thank you github for the github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Can I contribute to the project?]**
+
+  - [Yes, we welcome contributions from the open source community]
+
+- **[How often is the code base update]**
+
+  - [Frequently, we review the code and welcome requests for changes from the community]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Obimbo07/To-do-list/blob/main/LICENSE) licensed.
```

Comment:
- [ ] Please refactor your `path` for the license, it seems like you are giving a `absolute` path for your `LICENSE` file, which is risky and make a separate request for to open the LICENSE file, even if a person is reading the README file locally. Kindly follow the below suggestion to tackle with this problem.

```suggestion
This project is [MIT](./LICENSE) licensed.
```